ngraph/frontend/paddlepaddle/include/paddlepaddle_frontend/place.hpp

```diff
@@ -170,7 +170,7 @@ namespace ngraph
             Ptr get_target_tensor(const std::string& outputName) const override;
             Ptr get_target_tensor(const std::string& outputName,
                                   int outputPortIndex) const override;
-
+            int state = 0; // used to determine cut nodes
```

Comment:
Why did you add a class member to public section?

Reply:
removed